Mathlib/Topology/Algebra/Module/WeakDual.lean

```diff
@@ -314,6 +314,45 @@ instance instTopologicalSpace : TopologicalSpace (WeakSpace 𝕜 E) :=
 instance instContinuousAdd : ContinuousAdd (WeakSpace 𝕜 E) :=
   WeakBilin.instContinuousAdd (topDualPairing 𝕜 E).flip
 
+
+/-- There is a canonical map `E → WeakSpace 𝕜 E` (the "identity"
+mapping). It is a linear equivalence. -/
+def toWeakSpace : E ≃ₗ[𝕜] WeakSpace 𝕜 E := LinearEquiv.refl 𝕜 E
+-- align?
```

Comment:
#align is for theorems/definitions that come from mathlib3. There is no need for #align for new theorems

Reply:
Thanks for your comments!
I didn't know about "align", I just tried to use similar codes in Weak dual of normed space [#](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/NormedSpace/WeakDual.html#Weak-dual-of-normed-space) so I left these "align"s. I will delete them.

Reply:
I deleted all aligns.